articles/machine-learning/how-to-auto-train-image-models.md

```diff
@@ -359,14 +359,49 @@ In individual trials, you directly control the model architecture and hyperparam
 Instance segmentation | **MaskRCNN ResNet FPN**| `maskrcnn_resnet18_fpn` <br> `maskrcnn_resnet34_fpn` <br> ***`maskrcnn_resnet50_fpn`\****  <br> `maskrcnn_resnet101_fpn` <br> `maskrcnn_resnet152_fpn`
 
+#### Supported model architectures - HuggingFace and MMDetection (preview)
+
+With the new backend that runs on [AzureML Pipelines](concept-ml-pipelines.md), you can additionally use any image classification model from the [HuggingFace Hub](https://huggingface.co/models?pipeline_tag=image-classification&library=transformers) which is part of the transformers library (such as microsoft/beit-base-patch16-224), as well as any object detection or instance segmentation model from the [MMDetection Version 2.28.2 Model Zoo](https://mmdetection.readthedocs.io/en/v2.28.2/model_zoo.html) (such as atss_r50_fpn_1x_coco). 
+
+In addition to supporting any model from HuggingFace Transfomers and MMDetection 2.28.2, we also offer a list of curated models from these libraries in the azureml-staging registry. These curated models have been tested thoroughly and use default hyperparameters selected from extensive benchmarking to ensure effective training. The table below summarizes these curated models.
+
+Task |  model architectures | String literal syntax
+---|----------|----------
+Image classification<br> (multi-class and multi-label)| **BEiT**: A Vision Transformer (ViT), which is a transformer encoder model (BERT-like). <br> **ViT**: A BERT-like transformer encoder model which is pretrained on a large collection of images in a supervised fashion. <br> **DeiT**: A more efficiently trained Vision Transformer (ViT) <br> **SwinV2**: A type of Vision Transformer used in both image classification and dense recognition tasks | `microsoft/beit-base-patch16-224-pt22k-ft22k`<br> `google/vit-base-patch16-224`<br> `facebook/deit-base-patch16-224`<br> `microsoft/swinv2-base-patch4-window12-192-22k`
+Object Detection | TODO |  `sparse_rcnn_r50_fpn_300_proposals_crop_mstrain_480-800_3x_coco `<br> `sparse_rcnn_r101_fpn_300_proposals_crop_mstrain_480-800_3x_coco` <br> `deformable_detr_twostage_refine_r50_16x2_50e_coco` <br> `vfnet_r50_fpn_mdconv_c3-c5_mstrain_2x_coco` <br> `vfnet_x101_64x4d_fpn_mdconv_c3-c5_mstrain_2x_coco` <br> `yolof_r50_c5_8x8_1x_coco` <br> `mask_rcnn_swin-t-p4-w7_fpn_fp16_ms-crop-3x_coco`
+Instance Segmentation | TODO | TODO
+
+We constantly update the list of curated models. You can get the most up to date list of the curated models for a given task using the Python SDK:
+```
+credential = DefaultAzureCredential()
+ml_client = MLClient(credential, registry_name="azureml-staging")
```

Comment:
why staging?

Reply:
azureml-staging is the registry where the models will be for public preview